TM1py/Exceptions/Exceptions.py

```diff
@@ -24,5 +25,9 @@ def reason(self):
     def response(self):
         return self._response
 
+    @property
+    def headers(self):
+        return self._headers
+
     def __str__(self):
         return "Text: {} Status Code: {} Reason: {}".format(self._response, self._status_code, self._reason)
```

Comment:
I think it would make sense to adjust the `__str__` method of this class. So it would look kinda like this:
``` python
  def __str__(self):
        return "Text: {} Status Code: {} Reason: {} Headers: {}".format(
            self._response,
            self._status_code,
            self._reason,
            self._headers)
```

Reply:
Sounds good I will make an update and commit again